http/file_server_test.ts

```diff
@@ -970,3 +972,45 @@ Deno.test(
     await Deno.remove(tempDir);
   },
 );
+
+Deno.test("file_server prints local and network urls", async () => {
+  const port = await getAvailablePort();
+  const cmd = new Deno.Command(Deno.execPath(), {
+    args: [
+      "run",
+      "--no-lock",
+      "--quiet",
+      "--allow-net",
+      "--allow-read",
+      "--allow-sys=networkInterfaces",
+      "--config",
```

Comment:
Reduces flakiness (ref #4168):
```suggestion
      "--config",
      "--no-lock",
```

Reply:
`--no-lock` is at the above